pkg/controller/notification/notification_controller_test.go

```diff
@@ -0,0 +1,156 @@
+package notification
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/host-operator/pkg/apis"
+	"github.com/codeready-toolchain/host-operator/pkg/configuration"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	murtest "github.com/codeready-toolchain/toolchain-common/pkg/test/masteruserrecord"
+
+	"github.com/spf13/cast"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	apiv1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func TestNotificationSuccess(t *testing.T) {
+	// given
+	restore := test.SetEnvVarAndRestore(t, "HOST_OPERATOR_DURATION_BEFORE_NOTIFICATION_DELETION", "10s")
+	defer restore()
+
+	t.Run("will not do anything and return requeue with shorter duration that 10s", func(t *testing.T) {
```

Comment:
Will not do anything in case of what?
This is not very informative.
A good description would tell what use case you are testing here.
Something like "sent notification is requeued with the remaining deletion timeout"

Reply:
this is actually copy/pasted from an older pr for change tier requests... so i thought the test name was fine. i will fix it in both places..